### PR TITLE
Temper Protection

### PR DESCRIPTION
**Pull request by joinimran**
Updated the correct information as Windows Temper protection was released in Windows 10 v 1903 and later. Made the necessary changes.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7150

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by denisebmsft** [comment]
@joinimran I will doublecheck this, but I tamper protection is supported for a few previous versions of Windows 10 (as in 1709 and later). You won't see it in the Windows Security app for 1709, 1803, or 1809, but that does not mean tamper protection isn't supported for those versions of Windows 10. I am going to hold this pull request until I get confirmation from our PM on this. Thanks for the submission!

**Comment by joinimran** [comment]
Thanks for clarification @denisebmsft - I have gone through different security-related blogs to get this confirmed. But this a great idea to double-check with PM before merging this PR. 

Thanks.

**Comment by illfated** [comment]
@joinimran : Thank you for your work on this PR. May I ask you to note the difference between "temper" and *tamper* in English, and then edit your PR title to say **Tamper Protection** instead? 😉

- tamper: weaken, make a negative change, or meddle with
- temper: anger, passion, or bad mood

File: windows/security/threat-protection/microsoft-defender-antivirus/prevent-changes-to-security-settings-with-tamper-protection.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ You must have appropriate [permissions](../microsoft-defender-atp/assign-portal-
...
     - Your organization uses [Intune to manage devices](https://docs.microsoft.com/intune/fundamentals/what-is-device-management). ([Intune licenses](https://docs.microsoft.com/intune/fundamentals/licenses) are required; this is included in Microsoft 365 E5.)
-    - Your Windows machines must be running Windows 10 OS [1709](https://docs.microsoft.com/windows/release-information/status-windows-10-1709), [1803](https://docs.microsoft.com/windows/release-information/status-windows-10-1803), [1809](https://docs.microsoft.com/windows/release-information/status-windows-10-1809-and-windows-server-2019) or later. (See [Windows 10 release information](https://docs.microsoft.com/windows/release-information/) for more details about releases.)
+    - Your Windows machines must be running Windows 10, version [1903](https://docs.microsoft.com/en-us/windows/release-information/status-windows-10-1903) or later. (See [Windows 10 release information](https://docs.microsoft.com/windows/release-information/) for more details about releases.)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-antivirus/prevent-changes-to-security-settings-with-tamper-protection.md:93]:
```suggestion
    - Your Windows machines must be running Windows 10, version [1903](https://docs.microsoft.com/windows/release-information/status-windows-10-1903) or later. (See [Windows 10 release information](https://docs.microsoft.com/windows/release-information/) for more details about releases.)
```

**Comment by joinimran** [comment]
Thanks @illfated to sharing the difference. Will make a note of this.

**Comment by e0i** [comment]
@denisebmsft Has there been any updates from PM regarding the proposed changes? Thank you.

**Pull request closed without merging**